### ISSUE TITLE
UI improvement suggest

### ISSUE THREAD

**Issue by z826540272** [Maassoft/ColorControl#26]
can you make it look like this ?
![110.png](https://i.loli.net/2021/04/21/uITA7GfjEDchUXZ.png)

**Comment by z826540272** [comment]
OR like this
![1111.png](https://i.loli.net/2021/04/21/QR1p4YHoxN3aVuT.png)

**Comment by Maassoft** [comment]
Yes, I can look if I can embed the remote control window within the main window. Maybe that you must activate it by clicking on a button a single time, but after that will stay visible without clicking on the button again (sort of toggle).
I see you have an AMD graphics card? Have you tried using the settings in the AMD controller? I would like to know if switching HDR on AMD works, my old HD 7950 does not support HDR.

**Comment by z826540272** [comment]
amd HDR toggle works

**Comment by Maassoft** [comment]
Thanks for testing. 
I've now released a new version with a picture of a remote control and you can click most of the buttons.
Please check it out if you have the time.
